packages/smart-contracts/test/TestInvite.js

```diff
@@ -103,6 +103,7 @@ contract('ContactInvite', accounts => {
 
     await truffleAssert.fails(
       invites.retrieveStake(
+        accounts[0],
```

Comment:
Thanks for reverting the contract changes, could you also revert these tests please if they were related to your previous updates.